src/main/java/com/github/igorperikov/hollow/HollowAPIGeneratorUtility.java

```diff
@@ -53,6 +55,9 @@ public static HollowAPIGenerator createHollowAPIGenerator(
         if (properties.restrictApiToFieldType) {
             builder.withRestrictApiToFieldType();
         }
+        if(!isNotBlank(properties.classPostfix)) {
```

Comment:
There is a double negation here which makes the code to not work as intended (I think). It should be:

`if(isNotBlank(properties.classPostfix))`

Reply:
yes good catch! Too many attempts at avoiding the new dependency and not cleaning up properly.